doc/examples/cloud-config.txt

```diff
@@ -192,8 +192,8 @@ cloud_config_modules:
 
 # ssh_import_id: [ user1, user2 ]
 # ssh_import_id will feed the list in that variable to
-#  ssh-import-id, so that public keys stored in launchpad
-#  can easily be imported into the configured user
+#   ssh-import-id, so that public keys stored in launchpad
+#   can easily be imported into the configured user
```

Comment:
I think this line was accidentally indented before, so should probably be:

```suggestion
# ssh-import-id, so that public keys stored in launchpad
# can easily be imported into the configured user
```

Reply:
fixup 476303f